data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApi.java

```diff
@@ -284,6 +285,7 @@ private void addTekuSpecificHandlers(final DataProvider provider) {
     app.get(GetSszState.ROUTE, new GetSszState(provider, jsonProvider));
     app.get(GetStateByBlockRoot.ROUTE, new GetStateByBlockRoot(provider, jsonProvider));
     app.get(Liveness.ROUTE, new Liveness());
+    app.get(Readiness.ROUTE, new Readiness(provider));
```

Comment:
this will correspond to adding a test in `BeaconRestApiV1Test`.

if you look for `Liveness.ROUTE`, it's just basically another entry in the builder, and this ensures the route is actually hooked up (in case we're ever re-factoring and break it)
`builder.add(Arguments.of(Readiness.ROUTE, Readiness.class));`

Reply:
Thanks for the hint,
I was aware I had to update `BeaconRestApiV1Test `but I miss that one.
It is updated now.